src/main/resources/build.template.properties

```diff
@@ -19,6 +19,11 @@ app.region=us-central1
 # e.g. app.version = 8-0-0
 app.version = 8-0-0
 
+# This is the base URL for production environment
+# This will be the URL for both backend and frontend as they are not separately served for production
```

Comment:
This should be a known thing to many developers as static files are served from the server by default? Seems redundant to me.

Reply:
Thanks for the comment.

I added that here as it is mentioned at [Config.java](https://github.com/TEAMMATES/teammates/blob/master/src/main/java/teammates/common/util/Config.java#L216). I will consolidate other comments for this PR and fix all in one go later.

Reply:
I have updated the comments and replaced it with more useful information.